news/2025/2025-04-15-the-followpoint-azer-the-tournament-titan.md

```diff
@@ -0,0 +1,305 @@
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/2155578" style="background-image: url('/wiki/shared/avatars/Azer.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/2155578">Azer</a></p>
+
+Before osu!, I played a lot of shooter and MMO games, and was already doing content creation for several years already, so I'm sure I would've kept going in that direction. I played a lot of Call of Duty and Counter Strike, and I got really hooked on Overwatch when it initially released (I was semi pro for a year or two) so I think that would have been the most likely!
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+If you could meet any osu! player IRL, who would you want to meet the most, and why?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/2155578" style="background-image: url('/wiki/shared/avatars/Azer.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/2155578">Azer</a></p>
+
+Honestly this list is filled with tons of European players I never got to meet. I've been to most parts of NA and met a few Japanese players, but never got to go to [COE](/wiki/Community/cavoe's_osu!_event) or anything so I'd love to hangout with some other long time players from there, probably [Bubbleman](https://osu.ppy.sh/users/5182050) and [Rafis](https://osu.ppy.sh/users/2558286).
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+If osu! ever introduced a new, 5th game mode, what do you think it should be? What's your vision on what could potentially be made into said game mode?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/2155578" style="background-image: url('/wiki/shared/avatars/Azer.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/2155578">Azer</a></p>
+
+Considering that all the osu! modes take inspiration from other existing games, I feel like they all cover most of what's possible on PC hardware, so it's hard to say. If there's one thing that could be possible, it could be more of a visually focused experience like Geometry Dash or Groove Coaster, maybe? Outside of that, I think that most other interesting experiences require specific inputs that aren't really possible or intuitive on mouse/keyboard, so I doubt they would ever be attempted.
```

Comment:
```suggestion
Considering that all the osu! modes take inspiration from other existing games, I feel like they all cover most of what's possible on PC hardware, so it's hard to say. If there's one thing that might be possible, it could be more of a visually focused experience like Geometry Dash or Groove Coaster, maybe? Outside of that, I think that most other interesting experiences require specific inputs that aren't really possible or intuitive on mouse/keyboard, so I doubt they would ever be attempted.
```